docs/getting_started/preparation.md

```diff
@@ -1,23 +1,147 @@
 # Preparation
 
+Flowcean is a Python framework and as such, you need to setup a runtime environment to work with it.
+This section will cover the essential steps with hints for the more advanced users and a step-by-step explanation for beginners.
+
 ## Git
 
-!!! todo
+Git is a version control system, which you will need if you want to work with the source code of Flowcean or with any other source code.
+By now, Flowcean does not have a release on PyPi, i.e., you have to download the source code to use it.
+The recommended way is to use git for this.
+
+### Install on Windows
+
+User on windows can download the latest version on [https://git-scm.com/downloads](https://git-scm.com/downloads).
+Double-click on the installation file and keep the settings to their defaults (unless you know what you're doing).
+
+After the installation, open a Powershell and you should be able to type `git` into it and see the help page:
+
+```PS
+> git 
+Usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]
+        [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
+        [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
+        [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
+        [--config-env=<name>=<envvar>] <command> [<args>]
+```
+**TODO** Show results from an actual power shell
 
-    - download
-    - install
+### Install on Linux
+
+Git is shipped with the package manager of all common distributions and you can install it from the command line.
+See [here](https://git-scm.com/download/linux) to get the command for your distribution, e.g.:
+
+```bash
+sudo apt-get install git  # On Debian/Ubuntu
+sudo pacman -S git  # On Arch linux
+```
 
 ## Python
 
-!!! todo
+Flowcean is written in Python and requires at least version 3.12.
```

Comment:
I think we only need hatch, and hatch will take care of installing the appropriate python version... And for hatch there are even standalone installers. Have a look at https://hatch.pypa.io/latest/install/

Reply:
I have added a hint that hatch can be used and links to the documentation.  
The experienced user may use this and skip the rest of the preparation/installation.

For beginners, it is [advisable](https://www.bitecode.dev/p/why-not-tell-people-to-simply-use) to go the manual route, and we should provide the required steps here.

I don't use hatch and it works fine as well :-)

Reply:
How do we ensure to always keep the python version in the documentation up-to-date? We already specify that in the `pyproject.toml` ... If we upgrade to a newer version, we have to ensure our python version in the documentation is always in sync with the documentation. This is why I preferred to mention hatch, as this respects the metadata in the `pyproject.toml`. If we keep the explanation for the manual way, I would be in favor of just linking to the python version, such that we are not in need of updating it.